published/external/xdpapi.h

```diff
@@ -156,7 +228,269 @@ XdpUnloadApi(
     FreeLibrary(XdpHandle);
 }
 
-#endif // !defined(_KERNEL_MODE)
+#endif // defined(_KERNEL_MODE)
+
+#if defined(_KERNEL_MODE)
+
+#include <netioddk.h>
+#include <xdp/guid.h>
+
+#define POOLTAG_NMR_CLIENT      'cNdX'   // XdNc
```

Comment:
should be either parameterized, prefixed with a `_`, or `#undef`'d at the end of the header

Reply:
Could you elaborate the direction?
For me it is just a macro definition. what can be parameterized? `_` means `_POOLTAG_NMR_CLIENT` or `'_cNdX'`?